api/v1/configurationpolicy_types.go

```diff
@@ -120,8 +125,7 @@ func (e EvaluationInterval) GetNonCompliantInterval() (time.Duration, error) {
 type ConfigurationPolicySpec struct {
 	Severity           Severity           `json:"severity,omitempty"`          // low, medium, high
 	RemediationAction  RemediationAction  `json:"remediationAction,omitempty"` // enforce, inform
-	NamespaceSelector  Target             `json:"namespaceSelector,omitempty"`
-	LabelSelector      map[string]string  `json:"labelSelector,omitempty"`
```

Comment:
So, it looks like it might be in use for `cert-policy-controller` in order to select certificates by label. (Though even that isn't documented--I'll verify and open an issue to document it better if we need it.) 
That said, should the `LabelSelector` field be left in place here for now or write it off as being specific to `cert-policy-controller`? I'm leaning toward the latter.

Reply:
Since it's neither documented nor used by the configuration-policy-controller, I feel even better about just removing it. Leave it as specific to cert-policy. And we won't need to include it in the nucleus 😄 

I think it's also present in the iampolicy spec, but not used.